README.ru-ru.md

```diff
@@ -0,0 +1,241 @@
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>Недоступно</td>
+    </tr>
+  </tbody>
+</table>
+
+> **Примичание**:
+> Таблица выше служит для иллюстрации возможностей, доступных в Collapse сейчас. Этот список будет постоянно обновляться по мере добавления новых возможностей в Collapse и выхода новых игр. Если у вас возникли проблемы с какой-либо поддерживаемой возможностью, проверьте вкладку GitHub с проблемами, сообщите, если не нашли открытых проблем для неё.
+> > Пожалуйста, имейте в виду, что возможность конвертации региона сейчас доступна только для Honkai Impact: 3rd. Конвертация региона других игр miHoYo/Cognosphere Pte. Ltd. сейчас не в планах.
+
+# Загрузить последний релиз
+Для просмотра релизов [**нажмите сюда**](https://github.com/neon-nyan/CollapseLauncher/releases).
+
+# Системные требования
+- ОС: **Windows 10 версии 1809 (сборка 17763)** или выше / **Windows 11 (любые сборки)**
+- Архитектура: **x64/AMD64**
+- Доступ в интернет: **Есть**
+
+***
+
+# Инструкция для локальной компиляции/разработки
+> ### Больше информации Вы найдёте в [**Руководстве по внесению изменений**](https://github.com/neon-nyan/Collapse/blob/main/CONTRIBUTING.md)
+
+***
+
+# Сторонние репозитории и библиотеки, используемые в этом проекте
+- [**Windows UI Library**](https://github.com/microsoft/microsoft-ui-xaml) от Microsoft
+- [**Windows App SDK**](https://github.com/microsoft/WindowsAppSDK) от Microsoft
+- [**HDiffPatch**](https://github.com/sisong/HDiffPatch) от housisong
+- [**Color Thief .NET**](https://github.com/neon-nyan/ColorThief) от KSemenenko
+- [**SevenZipExtractor**](https://github.com/neon-nyan/SevenZipExtractor) от adoconnection
+- [**Hi3Helper.Http**](https://github.com/neon-nyan/Hi3Helper.Http) от neon-nyan
+- [**Hi3Helper.EncTool**](https://github.com/neon-nyan/Hi3Helper.EncTool) от neon-nyan
+- [**Crc32.NET**](https://github.com/neon-nyan/Crc32.NET) от force-net
+- [**UABT**](https://github.com/neon-nyan/UABT) от _unknown_
+
+**Отказ от ответственности**: Этот проект **НИ В КОЕМ СЛУЧАЕ НЕ СВЯЗАН** с [**miHoYo (miHoYo Co., Ltd.)**](https://www.mihoyo.com/) и [**HoYoverse (COGNOSPHERE PTE. LTD.)**](https://www.hoyoverse.com/en-us) также является полностью открытым. Любой вклад приветствуется! 😃
+
+# Обзор дизайна лаунчера
+![](https://raw.githubusercontent.com/neon-nyan/CollapseLauncher-Page/main/images/UI%20Overview%20RC2.webp)
+
+# Как я могу поддержать этот проект?
+Поддержка не является обязательной, но очень ценится и мотивирует нас вкладывать больше времени в проект и поддерживать его и другие проекты. Если Вы решили нас поддержать, вот способы которыми Вы можете это сделать :smile::
+- **[GitHub Sponsors](https://github.com/sponsors/neon-nyan)**
+- **[QRIS (Quick Response Code Indonesian Standard)](https://qris.id/homepage/)** (Coming Soon:tm:)
```

Comment:
We are ditching QRIS for mean to sponsor, so you can delete this line. I'll update it accordingly on other languages

Reply:
[Removed](https://github.com/Kajitsy/Collapse/commit/3ae63c16be83e1b462710cca1e3349cfea573b2f)